src/test/scala/com/typesafe/npm/NpmSpec.scala

```diff
@@ -15,7 +15,7 @@ class NpmSpec extends Specification with NoTimeConversions {
   def withEngine[T](block: ActorRef => T): T = {
     val system = ActorSystem("test-system", classLoader = Some(this.getClass.getClassLoader))
     try {
-      val engine = system.actorOf(Trireme.props(), "engine")
+      val engine = system.actorOf(Node.props(), "engine")
```

Comment:
@benmccann You also need to change this line in `npm-tester/src/main/scala/com/typesafe/npm/tester/Main.scala`:
```
-    val engine = system.actorOf(Trireme.props(), "engine")
+    val engine = system.actorOf(Node.props(), "engine")
```

You can test by switching to the `npm-tester` project and run it:
```
$ sbt
> projects
[info] 	   npm-tester
[info] 	 * root
> project npm-tester
> run
```